doc/classes/ProjectSettings.xml

```diff
@@ -2717,6 +2717,12 @@
 		<member name="xr/openxr/form_factor" type="int" setter="" getter="" default="&quot;0&quot;">
 			Specify whether OpenXR should be configured for an HMD or a hand held device.
 		</member>
+		<member name="xr/openxr/foveation_dynamic" type="bool" setter="" getter="" default="true">
+			If true and foveation is supported, will automatically adjust foveation level based on framerate (and [code]xr/openxr/foveation_level[/code] will be ignored).
```

Comment:
I'm not sure the "will be ignored" is true. From the spec and my own testing, the foveation level seems to set a maximum that will be used by the dynamic foveated rendering:

![Selection_076](https://github.com/godotengine/godot/assets/191561/64c5df15-6d3f-4630-b365-5534efe2d05e)

From: https://registry.khronos.org/OpenXR/specs/1.0/html/xrspec.html#XR_FB_foveation

Reply:
Ah I changed that based on feedback, I think its safer to follow what the spec says